tools/restatectl/src/commands/node/list_nodes.rs

```diff
@@ -46,18 +43,8 @@ pub struct ListNodesOpts {
 }
 
 pub async fn list_nodes(connection: &ConnectionInfo, opts: &ListNodesOpts) -> anyhow::Result<()> {
-    let channel = grpc_channel(connection.cluster_controller.clone());
-    let mut client =
-        ClusterCtrlSvcClient::new(channel).accept_compressed(CompressionEncoding::Gzip);
-
-    let mut response = client
-        .list_nodes(ListNodesRequest::default())
-        .await
-        .map_err(|e| anyhow::anyhow!("failed to list nodes: {:?}", e))?
-        .into_inner();
```

Comment:
Can we remove the `list_nodes` call if all it does is to give you the `NodesConfiguration`?